tests/apis/prv_api/test_service_areas.py

```diff
@@ -151,7 +152,7 @@ def test_all_polygons_get(client):
 def test_polygon_creation(client):
     response = client.post(
         POLY_BASE_URL,
-        data={"label": "test", "geom": MOCK_GEOJSON, "areas": []},
+        data={"label": "test", "geom": json.dumps(MOCK_GEOJSON), "areas": []},
```

Comment:
You can add a test with a simple polygon also to check we won't have a regression